server/handlers/error.go

```diff
@@ -113,6 +110,7 @@ const (
 	ErrGetMeshModelsCode                = "2257"
 	ErrGetUserDetailsCode               = "2258"
 	ErrResolvingRelationship            = "2259"
+	ErrGetLatestVersionCode             = ""
```

Comment:
Why is this empty?

Reply:
@Revolyssup This has been revised to use a random number, until I can clean up the duplicates.